resources/dicts/patrols/general/hunting.json

```diff
@@ -3691,5 +3691,241 @@
+            "exp": 0,
+            "weight": 20,
+	    "prey": ["very_small"],
+	    "stat_trait": ["childish", "oblivious", "strange", "troublesome"],
+	    "can_have_stat": ["r_c"],
+	    "relationships": [
+		{
+            	"cats_to": ["p_l"],
+            	"cats_from": ["s_c"],
+            	"mutual": true,
+            	"values": ["dislike"],
+            	"amount": 10
+		}
+	    ]
+            
+        },
+	{
+            "text": "{PRONOUN/p_l/poss} nerves already frayed, it only takes one more snide comment from r_c for p_l to explode, snarling in r_c's face the first insult that comes to mind. The two abruptly part ways, their hunting forgotten.",
+            "exp": 0,
+            "weight": 20,
+	    "prey": ["very_small"],
+	    "relationships": [
+		{
+            	"cats_to": ["p_l"],
+            	"cats_from": ["r_c"],
+            	"mutual": true,
+            	"values": ["dislike"],
+            	"amount": 10
+		}
+	    ]
+            
+        },
+	{
+            "text": "Already on edge from the constant bickering, when r_c makes a particularly hurtful remark, s_c retaliates, {PRONOUN/s_c/poss} claws scratching r_c's cheek. The pair silently return to camp - each cat seething - and refuse to tell any cat what happened.",
+            "exp": 0,
+            "weight": 20,
+	    "stat_trait": ["bloodthirsty", "fierce", "vengeful"],
+	    "relationships": [
+		{
+            	"cats_to": ["r_c"],
+            	"cats_from": ["s_c"],
+            	"mutual": true,
+            	"values": ["dislike"],
+            	"amount": 15
+		}
+	    ],
+	    "injury": [
+                {
+                  "cats": ["r_c"],
+                  "injuries": ["claw-wound"],
+                  "scars": ["CHEEK"],
+                  "no_results": false
+             
+                }
+              ],
+              "history_text": {
+                "scar": "m_c was scarred when a clanmate lashed out at {PRONOUN/m_c/object} while they were bickering on patrol.",
+                "reg_death": "m_c died after a small injury progressed to something worse.",
+                "lead_death": "died after a small injury progressed to something worse"
+              }
+            
+        }
+    ]
+},
+{
+    "patrol_id": "gen_hunt_group_competition",
+    "biome": ["any"],
+    "season": ["any"],
+    "types": ["hunting"],
+    "tags": [],
+    "patrol_art": "gen_speaking_cat_w1",
+    "min_cats": 3,
+    "max_cats": 6,
+    "min_max_status": {
+        "normal adult": [2, 6]
+    },
+    "weight": 20,
+    "intro_text": "Noticing how bored the patrol seems, r_c brightly suggests a hunting competition to add a bit of friendly rivalry to the outing.",
+    "decline_text": "The rest of the patrol dismiss the idea. Eventually, they give up on hunting as a group and head off to perform other tasks.",
+    "chance_of_success": 50,
+    "success_outcomes": [
```

Comment:
additional success outcomes you could add could be stat outcomes for speaker/mediator/hunter/charismatic/mediator/sense in various situations.  A speaker/charismatic cat encouraging others to join in, a hunter winning the competition, a mediator managing an argument over who won, a keen sense cat finding the most prey, ect.

Not required for PR acceptance, as I know it's a lot of work to add more outcomes, but a suggestion for further diversifying this patrol.

Reply:
Could also be fun to add some fails for things like impulsive cats blundering all over other hunters in their excitement, competitive cats could either ruin the hunt for others by being purposefully disruptive OR could excel and win the competition as a success.